test/Spec.hs

```diff
@@ -275,18 +275,23 @@ filterAnalyzedTicketsSpec =
                 \(ticketInfos :: [TicketInfo]) ->
                     length (filterAnalyzedTickets ticketInfos) `shouldBe` 0
 
-        it "should filter analyzed tickets" $
+        it "should filter analyzed tickets v1.0" $
             forAll (listOf $ genTicketWithFilteredTags ["analyzed-by-script-v1.0"]) $
                 \(ticketInfos :: [TicketInfo]) ->
                     length (filterAnalyzedTickets ticketInfos) `shouldBe` 0
 
+        it "should filter analyzed tickets v1.1" $
```

Comment:
Nice, even though the other scenario should be tested when testing - the one where all of the tickets have the tag, or some of the issues have the tag. Never mind for now.